simple/stats/importer.py

```diff
@@ -106,6 +111,30 @@ def _rename_columns(self) -> None:
 
     self.df = self.df.rename(columns=renamed)
 
+  def _write_observations(self) -> None:
+    # Melt dataframe so shape it similar to the observations table.
+    # Convert all values to str first, otherwise it inserts ints as floats.
+    observations_df = self.df.astype(str)
+    observations_df = observations_df.melt(
+        id_vars=[
+            constants.COLUMN_DCID, constants.COLUMN_DATE,
+            constants.COLUMN_PROVENANCE
+        ],
+        var_name=constants.COLUMN_VARIABLE,
+        value_name=constants.COLUMN_VALUE,
+    ).dropna()
+
+    # Reorder columns so they are in the same order as observations
```

Comment:
since import tool has full control, can just make the order correct at the first shot?

Reply:
We use the pandas `melt` method above (to unpivot the table) and it puts the variable and value columns at the end.